src/main/java/com/ait/lienzo/client/core/shape/toolbox/ToolboxVisibilityExecutors.java

```diff
@@ -28,8 +28,8 @@
 
 public class ToolboxVisibilityExecutors {
 
-    private static final double ANIMATION_SCALE_DELAY_MILLIS = 50;
-    private static final double ANIMATION_ALPHA_DELAY_MILLIS = 50;
+    static final double ANIMATION_SCALE_DELAY_MILLIS = 0d;
+    static final double ANIMATION_ALPHA_DELAY_MILLIS = 0d;
```

Comment:
@handreyrc @romartin Can we override the default value instead of changing it? Or maybe we can even delete animation code around Toolbox if we don't want to use it?

Reply:
@hasys I've checked the code and as we discussed it may require major efforts to refactor. We could open a new task for it.
Thanks